tests/integration/test_dynamodb.py

```diff
@@ -869,6 +869,75 @@ def _fetch_records():
         delete_table(table_name)
         kinesis.delete_stream(StreamName="kinesis_dest_stream")
 
+    def test_global_tables_version_2019(self):
+        # following https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/V2globaltables.tutorial.html
+
+        # create clients
+        dynamodb_us_east_2 = aws_stack.create_external_boto_client(
+            "dynamodb", region_name="us-east-2"
+        )
```

Comment:
can we test this against AWS?

you can create instrumented boto clients with the `create_boto_client` factory fixture
```python
def test_global_tables_version_2019(self, create_boto_client):
    dynamodb_us_east_2 = create_boto_client("dynamodb", region_name="us-east-2")
    #...
```